flake8_quotes/__init__.py

```diff
@@ -12,31 +12,53 @@ class QuoteChecker(object):
     QUOTES = {
         # When user wants only single quotes
         '\'': {
-            'good_single': '\'',
-            'good_multiline': '\'\'\'',
-            'bad_single': '"',
-            'bad_multiline': '"""',
+            'good': '\'',
+            'bad': '"',
         },
+
+        # When user wants only double quotes
         '"': {
-            'good_single': '"',
-            'good_multiline': '"""',
-            'bad_single': '\'',
-            'bad_multiline': '\'\'\'',
+            'good': '"',
+            'bad': '\'',
         },
     }
 
+    MULTILINE_QUOTES = {
```

Comment:
Main reason I dislike the current situation (also `QUOTES`) is that it contains redundant data. X's good is Y's bad and Y's good is X's bad. Also if there are really more quote styles, a list is handier as you could just remove the accepted style and have all bad ones. Especially with this patch using dicts doesn't really convey much more information apart from having a mapping from the setting value (e.g. `'`) to the actual value (`'''`). Also you do use a list already with `MULTILINE_QUOTES_STRINGS`.

But as it seems my patch ( #33 ) should be based on this one, I could implement it later.